docs/source/release-notes.html.md

```diff
@@ -7,6 +7,25 @@ weight: 8
 
 If you have any questions or comments about these notes, please contact DfE via Slack or email.
 
+## 28 June 2024
+
+We’ve added some new features to help providers ahead of the closure of the funding contract for the 2021 cohort on 31 July.
+
+The main impact will be ECTs and mentors with partial declarations currently assigned to the 2021 cohort being moved to the 2024 cohort.
+
+Because these ECTs and mentors are automatically shifted from a standard to an extended schedule, we’re allowing providers to change their schedules if necessary, using the existing [change-schedule](/api-reference/reference-v3.html#api-v3-participants-ecf-id-change-schedule-put) endpoint.
+
+Providers can also temporarily move participants back to the 2021 cohort if they need to make any final declarations before it closes. They must be returned to the 2024 cohort to start making submissions against that year.
+
+Note that the schedule and cohort cannot be changed when a migrated participant is in the 2024 cohort if there are existing declarations in any of the following states:
```

Comment:
This kind of contradicts the previous sentence, as they _can_ move them back to 2021 if they have those declarations (as long as the declarations are also in 2021)

Reply:
Just if I can understand this.. a LP can move participants back to 2021 if no declarations exist? So they can then make final declarations to those participants and then they must move them back to 2024 cohort alongside with the final declarations made? Is it allowed to change participants cohort with declarations?

Reply:
when a participant has declarations in a frozen cohort i.e. 2021, they can move those participants across cohorts and schedules normally. If for instance they have declarations in 2021, and got moved to 2024, then they want to add more declarations for that participant in 2021, they can move them back to 2021 (we didn't allow it before). If they declared against those participants in 2024 they will need to void those declarations before they can move back to 2021. Hope that answers your question

Reply:
so basically we treat a frozen cohort declarations as special and don't affect validations

Reply:
Tks @cwrw 👍 🔝